src/App.tsx

```diff
@@ -543,17 +662,26 @@ function App() {
               <div className="results-section">
                 <h3>wallet_sessionChanged results</h3>
                 <div className="notification-container">
-                  {walletSessionChangedResults ? (
-                    <details>
-                      <summary className="result-summary">
-                        {truncateJSON(walletSessionChangedResults).text}
-                      </summary>
-                      <code className="code-left-align">
-                        <pre id="wallet-session-changed-result">
-                          {JSON.stringify(walletSessionChangedResults, null, 2)}
-                        </pre>
-                      </code>
-                    </details>
+                  {walletSessionChangedHistory.length > 0 ? (
```

Comment:
length check not needed probably?

Reply:
used to show "No session changes detected" when no history is present

Reply:
oh i can't read